openpype/lib/delivery.py

```diff
@@ -0,0 +1,313 @@
+"""Functions useful for delivery action or loader"""
+import os
+from avalon import pipeline
+from avalon.vendor import filelink
```

Comment:
We must be sure this file won't be imported at any point of openpype initialization otherwise avalon import would cause crashes.